Llk/Parser.php

```diff
@@ -325,8 +325,11 @@ protected function _parse(Rule $zeRule, $next)
             }
 
             $namespace = $this->_tokenSequence->current()['namespace'];
+            $offset = $this->_tokenSequence->current()['offset'];
+
```

Comment:
To avoid a double call to `current()`, can you create a variable to hold the `$this->_tokenSequence->current()` value and use it for `$namespace` and `$offset` please?